src/Site/views/shared/container/container.html.twig

```diff
@@ -9,17 +9,15 @@
 
     <title>{{ app.website.name }}</title>
 
-    <link rel=stylesheet href="/dist/css/languageforge.css" />
+    <link rel=stylesheet href="/dist/css/languageforge.css?v={{ version }}" />
```

Comment:
This looks like an important change to me - with the version string implementing the cache .  Well done.

Reply:
yeah, that one might've been a big deal 😬  you're welcome though.